app/config.go

```diff
@@ -296,6 +321,10 @@ func (a *App) ClientConfigWithComputed() map[string]string {
 	// by the client.
 	respCfg["NoAccounts"] = strconv.FormatBool(a.IsFirstUserAccount())
 	respCfg["MaxPostSize"] = strconv.Itoa(a.MaxPostSize())
+	installationDate, err := a.getSystemInstallDate()
+	if err == nil {
+		respCfg["InstallationDate"] = strconv.FormatInt(installationDate, 10)
```

Comment:
Should we always send down this field, even if we can't determine the date?

Reply:
I'm not 100% sure. We can return `null` explicitly, because any other value will have meaning and we don't want to lie to the user.

I think I going to initialize that value to null explicitly.

Reply:
Done! Finally I initialize it to empty string, because `respCfg` is a `map[string]string`